src/triage/cli.py

```diff
@@ -200,6 +213,7 @@ def experiment(self):
             "project_path": self.args.project_path,
             "config": config,
             "replace": self.args.replace,
+            "matrix_storage_class": self.matrix_storage_map[self.args.matrix_storage]
```

Comment:
very minor thing, but if you like, you could have `argparse` perform this look-up, by setting `type`:

```
...
type=self.matrix_storage_map.get,
...
```

Reply:
When I try this I get an error from argparse: 

`TypeError: unhashable type: 'dict'`

Reply:
hah well … goodbye

Reply:
Yeah there are a few ways to get that to work ... but it gets weirder and I guess the real way to apply to this example is with an `argparse.Action` subclass. That would be interesting and useful and not hard ... but I doubt you'd want to bother here.